sites/public/styles/overrides.scss

```diff
@@ -56,9 +56,41 @@
           border: none;
         }
 
+        .navbar-dropdown-container {
+          margin-left: -144px;
+          width: auto;
+          .navbar-dropdown {
+            .navbar-dropdown-item-container {
+              display: flex;
+            }
+            .navbar-dropdown-item {
+              @apply border-t;
```

Comment:
I believe you can do 1 @apply and add all the styles to the one apply like:
`@apply border-t border-gray-450 flex flex-1 items-center normal-case`

Reply:
Done.